pkg/function/config.go

```diff
@@ -44,31 +49,65 @@ func Load(ctx context.Context, dir string) (*Function, error) {
 	}
 
 	// Then attempt to find inngest.cue, the canonical reference.
-	cue := filepath.Join(abs, "inngest.cue")
-	if _, err := os.Stat(cue); err == nil {
-		// The cue file exists.
-		byt, err := os.ReadFile(cue)
-		if err != nil {
-			return nil, err
-		}
+	cue, byt, err := findFileUp(filepath.Join(abs, cueConfigName))
```

Comment:
I wonder if we're doing the traversal in `findFileUp`, should we search for both files at once and return the filename and contents to the `Load` function? Either way, the output is passed to `Unmarshal` so we could reduce duplicate code and directory traversal.

Reply:
Have refactored this slightly to check for both `.cue` and `.json` files in each dir as it goes.